requirements.txt

```diff
@@ -1,8 +1 @@
-certifi==2017.11.5
```

Comment:
all of these are dependencies of cloudflare. I'm just taking the latest with this change. 

It worked. Its not ideal see the draft PR: https://github.com/neighborhoods/CloudflareBackup/pull/3

for a forever solution.